Automattic/wp-calypso
# Update upgrade path so Akismet Free to Akismet Personal is set to

## Proposed Changes

This PR updates the Akismet upgrade paths so that when a user upgrades from Akismet Free, the Akismet Personal plan (aka the Pay What You Want plan) defaults to the average $36

NOTE: I also used this opportunity to improve the Akismet products, fix a typo, fix a bug, and add some tests for the upgrade button

## Testing Instructions

1. Checkout this branch & Start your local environment
2. Sandbox `public-api.wordpress.com`
3. **Make sure you are using the store sandbox**
4. Make sure you have no Akismet plans active on your account ( you can delete them from the Store Admin )
5. Go to http://calypso.localhost:3000/checkout/akismet/ak_free_yearly and complete the checkout
6. Head to http://calypso.localhost:3000/me/purchases and select your free Akismet plan
7. Click "Upgrade"
8. Make sure that the PWYW plan defaults to $36
![image](https://user-images.githubusercontent.com/65001528/229159902-4db8ec2a-2be3-491d-bcee-db576e7829f3.png)

## Pre-merge Checklist
